src/AccessibilityInsights.SharedUx/Settings/ConfigurationModel.cs

```diff
@@ -614,7 +614,7 @@ public static ConfigurationModel GetDefaultConfigurationModel(FixedConfigSetting
                 SelectionByMouse = true,
                 ShowWelcomeScreenOnLaunch = true,
                 AlwaysOnTop = true,
-                PlayScanningSound = false,
+                SoundFeedback = SoundFeedbackMode.Auto,
```

Comment:
Someone who is upgrading could have `PlayScanningSound` set to either true or false. Do both of these map to `SoundFeedbackMode.Auto`? If not, we should provide appropriate mapping inside the `LoadDataFromJSON` method. 

At some future time, we'll drop support for versions that use `PlayScanningSound` and will want to remove that key from any dictionaries that we read. It might be worth adding a TODO inside `LoadDataFromJSON` so we have a reminder

Reply:
That depends – there are (at least) two ways to look at that:

* The production release doesn't include #1445. Therefore, a `true` value is technically equivalent to our new `Auto` setting (play a sound only when the SR flag is set). Map `true` to `Auto` as not to change behaviour.
* I made #1445 because the old behaviour (play a sound only when the setting is enabled and the SR flag is set) seems counterintuitive and, as far as I can tell, isn't documented. Therefore, if a user explicitly set the "play scanning sound" option to `true`, they probably mean `Always` (i.e. they don't care whether a screen reader is running if they've explicitly enabled the option). Map `true` to `Always`.